manifests/m/Minecodes/azure/1.1.1/Minecodes.azure.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Minecodes.azure

```

```suggestion
PackageIdentifier: Minecodes.4zur3
```
This is best to match `Publisher.PackageName`